.github/workflows/triage-labelled.yml

```diff
@@ -48,7 +48,13 @@ jobs:
     # Skip in forks
     if: >
       github.repository == 'vector-im/element-android' &&
-      contains(github.event.issue.labels.*.name, 'X-Needs-Design')
+      contains(github.event.issue.labels.*.name, 'X-Needs-Design') &&
+      (contains(github.event.issue.labels.*.name, 'S-Critical') &&
+       (contains(github.event.issue.labels.*.name, 'O-Frequent') ||
+        contains(github.event.issue.labels.*.name, 'O-Occasional')) ||
+       contains(github.event.issue.labels.*.name, 'S-Major') &&
+       contains(github.event.issue.labels.*.name, 'O-Frequent') ||
+       contains(github.event.issue.labels.*.name, 'A11y'))
```

Comment:
This is quite hard to read and (and so to check), I would add some parenthesis to avoid confusion.

I read this:
> X-Needs-Design && (S-Critical && (O-Frequent || O-Occasional) || S-Major && O-Frequent || A11y)

And I think this will be clearer:

> X-Needs-Design && (S-Critical && (O-Frequent || O-Occasional) || (S-Major && O-Frequent) || A11y)